CONTRIBUTING.md

```diff
@@ -132,34 +197,31 @@ Now navigate to your browser and open
 congratulations – you're all set. Otherwise, let us know by opening a GitHub
```

Comment:
I think `let us know by opening a Github issue...` should be changed to `let us know by asking in the [Contributors Room](https://gitter.im/FreeCodeCamp/Contributors) on Gitter. There also might be an error in the console of your browser or in Bash / Terminal / Command Line that will help identify the problem.`